instrumentation/spring/spring-webmvc/spring-webmvc-3.1/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/webmvc/v3_1/filter/ServletFilterConfig.java

```diff
@@ -29,53 +29,43 @@ Filter servletFilter() {
     return new Filter() {
 
       @Override
-      public void init(FilterConfig filterConfig) throws ServletException {}
+      public void init(FilterConfig filterConfig) {}
 
       @Override
       public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
           throws IOException, ServletException {
         HttpServletRequest req = (HttpServletRequest) request;
         HttpServletResponse resp = (HttpServletResponse) response;
         ServerEndpoint endpoint = ServerEndpoint.forPath(req.getServletPath());
-        HttpServerTest.controller(
-            endpoint,
-            () -> {
-              resp.setContentType("text/plain");
-              switch (endpoint.name()) {
-                case "SUCCESS":
-                  resp.setStatus(endpoint.getStatus());
-                  resp.getWriter().print(endpoint.getBody());
-                  break;
-                case "QUERY_PARAM":
+        if (endpoint != null && endpoint != ServerEndpoint.NOT_FOUND) {
+          AbstractHttpServerTest.controller(
```

Comment:
static import here?

Reply:
not sure the static import is worth it if there is only 1 usage